### ISSUE TITLE
Model has fixed input size

### ISSUE THREAD

**Issue by taylorshin** [amaiya/ktrain#245]
I successfully fine-tuned DistilBert on SST-2 using ktrain and got a weights h5 file. When I inspect the h5 file with `Netron`, nothing seems inherently wrong because it's just a bunch of weights.

![Screen Shot 2020-09-04 at 2 52 35 PM](https://user-images.githubusercontent.com/10328411/92287374-74dc1a00-eebe-11ea-9283-fe8765770d35.png)

Then, I converted the h5 file into a TensorFlow Lite model with the following code:
```
model_name = 'distilbert-base-uncased'
config = DistilBertConfig(num_labels=2)
transformer_model = TFDistilBertForSequenceClassification.from_pretrained(model_name, config=config)
transformer_model.summary()
transformer_model.load_weights('tf_model.h5')
converter = tf.lite.TFLiteConverter.from_keras_model(transformer_model)
converter.experimental_new_converter = True
converter.optimizations = [tf.lite.Optimize.DEFAULT]
converter.target_spec.supported_ops = [tf.lite.OpsSet.TFLITE_BUILTINS, tf.lite.OpsSet.SELECT_TF_OPS]
tflite_model = converter.convert()
# Save the TF Lite model
with tf.io.gfile.GFile(out_file, 'wb') as f:
    f.write(tflite_model)
```
When I inspect the resulting TFLite model, I see that the input has a fixed size of `3 x 5` when it should be something generic like `batch x sequence`. Could this have something to do with the fact that in `ktrain` one needs to configure models with a dataset even before training?

![Screen Shot 2020-09-04 at 3 00 00 PM](https://user-images.githubusercontent.com/10328411/92287793-a99ca100-eebf-11ea-9aa5-7f9f4ee3e61f.png)

**Comment by amaiya** [comment]
Hello,

When I said that the model is "configured" by inspecting the data, I meant that the **output** shape (i.e., number of labels), metrics, loss functions, etc. are configured automatically based on the data (e.g., classification vs. regression, multilabel vs multiclass).  The `maxlen` parameter does affect the sequence length, though.

I'm not familiar with TFLite or Netron, so I don't know where the 3x5 shape would be coming from, but I don't think it's anything *ktrain* is doing specifically for the above reasons.

**Comment by amaiya** [comment]
Btw, if you saved a model with `learner.save_model('/tmp/my_model')` (or `predictor.save`), you can reload it using:
```python
transformer_model = TFDistilBertForSequenceClassification.from_pretrained('/tmp/my_model`)
```
You don't need to call `load_weights`, `config = DistilBertConfig(num_labels=2)`, etc. 

[This FAQ entry](https://github.com/amaiya/ktrain/blob/master/FAQ.md#how-do-i-obtain-the-word-or-sentence-embeddings-after-fine-tuning-a-transformer-based-text-classifier) shows an example of this.
